code-examples/missingNumber/src/MissingNumber.java

```diff
@@ -0,0 +1,33 @@
+import java.util.stream.IntStream;
+
+class MissingNumber {
+    public static void main(String[] args) {
```

Comment:
a `PSVM` has to be included in a Java project, but I'm wondering: is an empty `main` method like this acceptable, or is there a better way of dealing with having to include a main method but not really needing to use it? Because, in this case I'm using my Test class as entry point, and it needs the methods I created - I don't really need to use the `main` method. Hope that makes sense!